src/store/index.js

```diff
@@ -0,0 +1,11 @@
+import { createStore, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
+import rootReducer from './rootReducer';
+
+export default function configureStore(initialState = {}) {
+  return createStore(
+    rootReducer,
+    initialState,
+    applyMiddleware(thunk),
```

Comment:
Nice process @bevkololi. Consider adding this middleware that spits an error when you try to mutate the store.
[reduxImmutableStateInvariant](https://github.com/leoasis/redux-immutable-state-invariant/blob/master/README.md)
<img width="539" alt="screen shot 2018-10-30 at 22 18 44" src="https://user-images.githubusercontent.com/40663486/47744244-f0049180-dc91-11e8-87ed-0eaad756f859.png">

Reply:
@charisschomba When making recommendations of third-party libraries ensure that you state the value it adds to the project

Reply:
@PaulKariukiRimiru I think what @charisschomba  meant was that this third party library will help because one is not allowed to mutate one's state in the reducers. In order to change state in your app, you should always return a new instance of your state with the changes. It will therefore spit an error whenever one tries to do so. I think we'll need it.

Reply:
@bevkololi That is what I wanted @charisschomba to write in the recommendation